Core/NoOpRunner.Core/NoOpRunner.cs

```diff
@@ -56,6 +56,26 @@ public NoOpRunner(IConnectionManager connectionManager)
             Observers = new List<IObserver>();
         }
 
+        public async Task OnMapMoveLoopFired()
+        {
+            Player.OnMapMoveLoopFired((WindowPixel[,])PlatformsContainer.GetShapes().Clone());
+            
+            PowerUpsContainer.MoveWithMap();
+            
+            PlatformsContainer.MoveWithMap();
+            
+            await connectionManager.SendMessageToClient(new MessageDto()
+            {
+                MessageType = MessageType.PlatformsUpdate,
+                Payload = PlatformsContainer.GenerateSequel()
+            });
+        }
+        /// <summary>
+        /// Eh??? Useless?? ORAORAORAORAORAORA
```

Comment:
Yes, unused right now, the powerup generation logic won't be here anyways, can remove